protocols/migration/participants.py

```diff
@@ -7,6 +7,60 @@
 from protocols.migration import BaseMigration
 
 
+class MigrationParticipants100To104SNAPSHOT(BaseMigration):
+    old_model = participant_1_0_0
+    new_model = participant_1_0_4_SNAPSHOT
+
+    def migrate_cancer_participant(self, cancer_participant):
+        migrated_participant = self.new_model.CancerParticipant.fromJsonDict(cancer_participant.toJsonDict())
+
+        migrated_participant.yearOfBirth = 0
+
+        if cancer_participant.primaryDiagnosisDisease is not None:
+            migrated_participant.primaryDiagnosisDisease = [cancer_participant.primaryDiagnosisDisease]
+
+        if cancer_participant.primaryDiagnosisSubDisease is not None:
+            migrated_participant.primaryDiagnosisSubDisease = [cancer_participant.primaryDiagnosisSubDisease]
+
+        if cancer_participant.assignedICD10 is not None:
+            migrated_participant.assignedICD10 = [cancer_participant.assignedICD10]
+
+        migrated_participant.tumourSamples = self.migrate_tumour_samples(cancer_participant.tumourSamples)
+        migrated_participant.germlineSamples = self.migrate_germline_samples(cancer_participant.germlineSamples)
+
```

Comment:
Don't we need to migrate explicitly the MatchedSamples too?

Reply:
Good point @priesgo I'll make sure to add an explicit test

Reply:
So the line:
```
migrated_participant = self.new_model.CancerParticipant.fromJsonDict(cancer_participant.toJsonDict())
```
does the MatchedSamples migration for us.